docs: `enscli` and `ensskills` docs (#2112)
namehash/ensnode: docs/ensnode.io/src/content/docs/docs/integrate/integration-options/ensskills.mdx

---
title: ensskills
description: Coming soon — skill bundles that give AI agents an opinionated contract for ENS.
---

import { Aside, LinkCard } from "@astrojs/starlight/components";

<Aside type="caution" title="Coming soon">

> **`ensskills`** is a planned collection of agent skills for ENS development and ENS lookups. The npm name is reserved; we're still shaping the design and no skills ship yet.

Reviewer: ```suggestion
**`ensskills`** is a planned collection of agent skills for ENS development and ENS lookups. The npm name is reserved; we're still shaping the design. `ensskills` will ship soon.
```
